Servers/manage-logs.sh

```
@@ -0,0 +1,204 @@
+        list_tenants
+        return 1
+    fi
+
+    local tenant_dir="$LOG_BASE_DIR/$tenant_id"
+    if [ ! -d "$tenant_dir" ]; then
+        echo "Tenant '$tenant_id' not found."
+        return 1
+    fi
+
+    mkdir -p "$backup_dir"
+    local backup_file="$backup_dir/${tenant_id}_logs_$(date +%Y%m%d_%H%M%S).tar.gz"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)